cmd/api/insert.go

```diff
@@ -11,86 +14,94 @@ import (
 	"github.com/tidwall/gjson"
 )
 
-const TABLE_NAME_HEADER = "X-SCRATCHDB-TABLE"
-const TABLE_NAME_QUERY = "table"
-const TABLE_NAME_JSON = "table"
-
-const FLATTEN_HEADER = "X-SCRATCHDB-FLATTEN"
-const FLATTEN_QUERY = "flatten"
-const FLATTEN_JSON = "flatten"
+const TableNameHeader = "X-SCRATCHDB-TABLE"
+const TableNameQuery = "table"
+const TableNameJson = "table"
 
-func (a *API) getTableName(c *fiber.Ctx) (string, string) {
-	if c.Get(TABLE_NAME_HEADER) != "" {
-		return utils.CopyString(c.Get(TABLE_NAME_HEADER)), "header"
-	}
-
-	if c.Query(TABLE_NAME_QUERY) != "" {
-		return utils.CopyString(c.Query(TABLE_NAME_QUERY)), "query"
-	}
-
-	return gjson.GetBytes(c.Body(), TABLE_NAME_JSON).String(), "body"
-}
+const FlattenHeader = "X-SCRATCHDB-FLATTEN"
+const FlattenQuery = "flatten"
+const FlattenJson = "flatten"
 
 func (a *API) getFlattenType(c *fiber.Ctx) (string, string) {
-	if c.Get(FLATTEN_HEADER) != "" {
-		return utils.CopyString(c.Get(FLATTEN_HEADER)), "header"
+	if c.Get(FlattenHeader) != "" {
+		return utils.CopyString(c.Get(FlattenHeader)), "header"
 	}
 
-	if c.Query(FLATTEN_QUERY) != "" {
-		return utils.CopyString(c.Query(FLATTEN_QUERY)), "query"
+	if c.Query(FlattenQuery) != "" {
+		return utils.CopyString(c.Query(FlattenQuery)), "query"
 	}
 
-	return gjson.GetBytes(c.Body(), FLATTEN_JSON).String(), "body"
+	return gjson.GetBytes(c.Body(), FlattenJson).String(), "body"
 }
 
 func (a *API) Insert(c *fiber.Ctx) error {
-	if c.QueryBool("debug", false) {
```

Comment:
Please add this back. The purpose is to allow someone to choose to debug a single request if they want.